src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,42 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringTest {
+    @DisplayName("문자열 \"1,2\" split")
+    @Test
+    void split() {
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat("1,2".split(",")).containsExactly("1", "2");
```
테스트 단위가 적어서 임시변수없이 바로 사용 해도 될것 같습니다.